app/src/main/java/com/xmartlabs/daydreaming/ui/LoginRegisterFragment.java

```diff
@@ -86,16 +89,29 @@ void onClickedLogin() {
 
   @OnClick(R.id.create_login_textview)
   void onClickedCreateLogin() {
+    setupErrorFieldsAsInvisible();
     //TODO create user and login user
     if (isRegisterSelected) {
       if (passwordFieldView.getText().toString().length() < 5) {
```

Comment:
I left it like this because the login/register/logout with a server was left out of the main scope.